Emby.Dlna/Profiles/SonyPs4Profile.cs

```diff
@@ -94,7 +94,7 @@ public SonyPs4Profile()
                 {
                     Container = "ts",
                     VideoCodec = "h264",
-                    AudioCodec = "mp3",
+                    AudioCodec = "aac,ac3",
```

Comment:
No mp2?

Reply:
I put MP2 at the end now, I have seen it in some LiveTV.
Since AAC is supported, there is no reason to transcode to MP2.